modules/api/pkg/handler/v2/routes_v2.go

```diff
@@ -7830,6 +7853,51 @@ func (r Routing) deleteClusterBackupSchedule() http.Handler {
 	)
 }
 
+func (r Routing) listProjectCBSL() http.Handler {
```

Comment:
From my perspective, swagger route comments are missing. See below for an example. I have to admit, I am not sure why they are missing for the already commited code for ClusterBackUp from above. But from my perspective the comments need to be there, so the routes are correctly displayed in swagger.

If we want to intentionally hide these routes from swagger, then I would like us to have a short comment in code that says so.